plugins/catalog-backend-module-gitlab/src/lib/types.ts

```diff
@@ -201,6 +201,11 @@ export type GitlabProviderConfig = {
    * If the project is a fork, skip repository
    */
   skipForkedRepos?: boolean;
+  /**
+   * List of repositories to exclude from discovery, should be the full path to the repository, e.g. `group/project`
+   * Paths should not start or end with a slash.
+   */
+  excludeRepos: string[];
```

Comment:
Let's make this optional just to be safe :pray:

Reply:
Sorry, I'm still feeling my way around TypeScript. I'm assuming I just need the `?` suffix to the property name, which I've now added. Thanks